alembic/versions/b39c755048eb_change_tables_to_numeric_column_type.py

```diff
@@ -0,0 +1,31 @@
+"""Change tables to Numeric column type
+
+Revision ID: b39c755048eb
+Revises: 
+Create Date: 2016-07-22 23:57:08.222835
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b39c755048eb'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import sideboard
+from sideboard.lib.sa import UTCDateTime, UUID, CoerceUTF8 as UnicodeText
+import uber
+from uber.config import c
+from uber.models import Column, Choice, MultiChoice
+
+
+def upgrade():
+    op.alter_column('group', 'tables', type_=sa.Numeric)
+    op.alter_column('group', 'tables', server_default="0")
```

Comment:
We have to do this alteration expressly after changing the column type, or SQLAlchemy will set the default as a double decimal '0' instead of a numeric '0'. Same deal with the downgrade. On a related note, this is why I've started checking the database manually after writing an upgrade rather than just trusting it to work, i.e.:
```
sudo su - postgres
psql
> \c rams_db
> \d group
```